.github/workflows/build.yml

```diff
@@ -0,0 +1,169 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  bygge_frontend:
+    runs-on: ubuntu-latest
+    outputs:
+      short_sha: ${{ steps.parse_commit.outputs.short_sha }}
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: v1-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            v1-yarn-
+      - run: yarn install
+      - run: yarn test
+      - run: yarn build
+      - id: parse_commit
+        run: echo "::set-output name=short_sha::$(git rev-parse --short HEAD)"
+      - uses: actions/upload-artifact@v1
+        with:
+          name: melosys-kodeverk
+          path: dist
+
+  bygge_backend:
+    needs: bygge_frontend
+    runs-on: ubuntu-latest
+    container: navikt/melosys-kodeverk-java:FIXME-ny-docker-tag
+    steps:
+      # GitHub Actions tillater kun filer under workspace i cache-nøkler
+      - run: cp /app.jar .
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: v1-m2build-${{ hashFiles('app.jar') }}
+          restore-keys: |
+            v1-m2build-
+      - uses: actions/download-artifact@v1
+        with:
+          name: melosys-kodeverk
+          path: melosys-kodeverk-java/melosys-kodeverk/dist
+      - name: Generere kildekode for Java-kodeverk
+        working-directory: melosys-kodeverk-java
+        run: |
+          set -e
+          java -jar ../app.jar melosys-kodeverk/dist/kodemap.yml
+      - name: Bygge Java-kodeverk
+        working-directory: melosys-kodeverk-java/melosys-internt-kodeverk
+        run: mvn clean install -B -V
+      - uses: actions/upload-artifact@v1
+        with:
+          name: melosys-kodeverk-java
+          path: melosys-kodeverk-java
+
+  tagge_frontend:
+    if: github.ref == 'refs/heads/master'
+    needs: bygge_backend
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - run: |
+          # FIXME: Bytt ut med en action for å tagge og committe
```

Comment:
Dette er bare tilpassa fra CircleCI-workflowen. Det funker, men er veldig grisete. Finnes sikkert en fiks ferdig action for å gjøre det samme, evt. kan vi legge en Javascript action i repoet som bruker Octokit API-et.

Reply:
Bruker `ad-m/github-push-action`, som også brukes i `melosys-schema`.